packages/dashboard/src/migration/convert-monitor-to-app-defintion.ts

```diff
@@ -0,0 +1,509 @@
+import {
```

Comment:
this file is pretty big! would love if it could be split up into smaller files (for example: convertLineChart, convertKPI)

Reply:
+1

Reply:
I think it makes sense to leave all the conversion logic in one file. It all gets used by convertWidget which in turn in used by migrateDashboard. I think a better way is to split by conversion logic, and functions that set defaults. Happy to do it the other way though if we don't agree.

Reply:
I could maybe change the file name from just getters to widget-data-getters to be more specific though?

Reply:
merged in for now, but i can change this on the follow up PR with the overlapping bug fix